src/Aspire.Hosting/DistributedApplicationOptions.cs

```diff
@@ -62,8 +66,7 @@ public DistributedApplicationOptions()
 
     private string? ResolveProjectDirectory()
     {
-        var assemblyMetadata = Assembly?.GetCustomAttributes<AssemblyMetadataAttribute>();
-        return GetMetadataValue(assemblyMetadata, "AppHostProjectPath");
+        return Assembly?.GetCustomAttribute<AssemblyConfigurationAttribute>()?.Configuration;
```

Comment:
I think this is the wrong method.

Reply:
Yeah, realized just after I posted in chat